components/picker/index.css

```diff
@@ -254,6 +251,11 @@ governing permissions and limitations under the License.
   &:active {
     background-color: var(--highcontrast-picker-background-color, var(--mod-picker-background-color-active, var(--spectrum-picker-background-color-active)));
     border-color: var(--highcontrast-picker-border-color-default, var(--mod-picker-border-active, var(--spectrum-picker-border-color-active)));
+    transform: perspective(max(
+        var(--spectrum-downstate-height, var(--mod-picker-block-size, var(--spectrum-picker-block-size))),
+        var(--spectrum-downstate-width, var(--mod-picker-inline-size, var(--spectrum-picker-inline-size))) * var(--spectrum-component-size-width-ratio-down)
+      ))
```

Comment:
I'm not sure about this use of custom properties for the width and height. If a user sets the `--mod-picker-inline-size` to `100%`, that'll mess up the math there and won't show a down state. Should I just leave out the `--mod` custom properties from the fallback chain? Or not have a default at all? Or something else?

Reply:
Hmm yeah this is tricky. Maybe an additional mod specific to downstate size at the front of the chain?